cmd/kube-controller-manager/app/options/options.go

```diff
@@ -285,6 +288,16 @@ func (s *KubeControllerManagerOptions) Flags(allControllers []string, disabledBy
 	fs := fss.FlagSet("misc")
 	fs.StringVar(&s.Master, "master", s.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig).")
 	fs.StringVar(&s.Generic.ClientConnection.Kubeconfig, "kubeconfig", s.Generic.ClientConnection.Kubeconfig, "Path to kubeconfig file with authorization and master location information (the master location can be overridden by the master flag).")
+
+	if !utilfeature.DefaultFeatureGate.Enabled(featuregate.Feature(clientgofeaturegate.WatchListClient)) {
+		if err := utilfeature.DefaultMutableFeatureGate.OverrideDefault(featuregate.Feature(clientgofeaturegate.WatchListClient), true); err != nil {
+			// it turns out that there are some integration tests that start multiple control plane components which
```

Comment:
This is going to be a bit problematic with compatibility version. Can I ask which tests are affected?

Reply:
It was `TestComponentSecureServingAndAuth` but let me check if anything has changed.

Reply:
Still failing - https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/128053/pull-kubernetes-integration/1846076203430055936